README.md

```diff
@@ -1,7 +1,5 @@
 # PostgreSQL logical replication Terraform provider
 
-[![Build Status](https://travis-ci.com/form3tech-oss/terraform-provider-postgresreplication.svg?branch=master)](https://travis-ci.com/form3tech-oss/terraform-provider-postgresreplication)
```

Comment:
If we want to keep it, one should be available, with branch filtering to only show the main branch: [Adding a workflow status badge](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge)

Reply:
Thanks for the link. I've added it back in!